### PR TITLE
QKS in Qiskit

### PR DESCRIPTION
**Pull request by albelew**

### CODE REVIEW

File: Experiments/QKS/2019_03_05_qiskit_implementation/main.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+IBMQ.load_accounts()
+
+# Check the connection for IBM Q
+try:
+    IBMQ.load_accounts()
+except:
+    print("""WARNING: There's no connection with the API for remote backends.
+             Have you initialized a file with your personal token?
+             For now, there's only access to local simulator backends...""")
+
```
Comment (mstechly) [inline Experiments/QKS/2019_03_05_qiskit_implementation/main.py]: But `main.py` doesn't actually run anything. Shouldn't it run `qks`?

File: Experiments/QKS/2019_03_05_qiskit_implementation/main.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+def main():
+    X_train, X_test, y_train, y_test = make_frames(
+        train_size=200, test_size=50, outer_length=2, inner_length=1
+    )
+
+    n_episodes = 20
+    scale = 1
+    n_trials = 1000
+    q = 2
```
Comment (mstechly) [inline Experiments/QKS/2019_03_05_qiskit_implementation/main.py]: `q` is not particularly good variable name - please change it to something more descriptive, like `number_of_qubits`.

File: Experiments/QKS/2019_03_05_qiskit_implementation/main.py [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    plt.title(title)
+    plt.scatter(data[:0], data[:1], s=5, c=target)
+    plt.savefig(name)
+
+    qc = make_qc(q, noisy)
+    p = X_train.shape[-1]
+
+    r = 1 if p / q < 1 else int(p / q)
+
+    QKS = QuantumKitchenSinks(qc, n_episodes=n_episodes, r=r, scale=scale, distribution='normal', n_trials=1000)
```
Comment (mstechly) [inline Experiments/QKS/2019_03_05_qiskit_implementation/main.py]: This kind of logic shouldn't be inside `make_plot` function, rather in some separate function, e.g. `run_qks` or sth. like that.

**Comment by zackgow** [comment]
> It looks good, just these small fixes.
> @zackgow - anything else?

Looks good to me, though I think a `report.md` file should be included to show that it leads to the same results. We'll likely need to make a few changes here so it'll work with MNIST, but I'm still running some experiments with that.

**Comment by mstechly** [comment]
@zackgow good point with the report!
@albelew once you fix this I will merge it, then we will reproduce results from Zack's experiment, add a raport and think what's next :)

**Comment by albelew** [comment]
@mstechly 
I've just added fixes to the code.

File: Experiments/QKS/2019_03_05_qiskit_implementation/main.py [hunk 4]
```diff
@@ -23,6 +24,16 @@ def logistic_regression(X_train, X_test, y_train, y_test):
 
     return train_preds, test_preds
 
+
+def run_qks(p, number_of_qubits, qc, n_episodes, scale, n_trials):
```
Comment (mstechly) [inline Experiments/QKS/2019_03_05_qiskit_implementation/main.py]: I have no idea what `p` and `r` mean - please use some more descriptive variable names.

File: Experiments/QKS/2019_03_05_qiskit_implementation/main.py [hunk 5]
```diff
@@ -76,6 +87,8 @@ def main():
              Have you initialized a file with your personal token?
              For now, there's only access to local simulator backends...""")
 
+main()
```
Comment (mstechly) [inline Experiments/QKS/2019_03_05_qiskit_implementation/main.py]: I think this happened to be here by accident.

File: Experiments/QKS/2019_03_05_qiskit_implementation/main.py (hunk 4 above)
Comment (mstechly) [reply Experiments/QKS/2019_03_05_qiskit_implementation/main.py]: Now I remember - `p` and `r` were some parameters of QKS, notation used in the paper.
Please add some documentation to this function, describing what's what + indicating that this reflects the notation from the paper.

**Comment by albelew** [comment]
@mstechly Added comments and deleted this "bug".

**Pull request merged**